website/content/docs/concepts/filtering/worker-tags.mdx

```diff
@@ -140,13 +140,6 @@ Once workers have tags, you can use these tags to control which
 workers are allowed to manage a given session by specifying optional worker filter attributes
 when you configure [targets](/boundary/docs/concepts/domain-model/targets).
 
-The `egress_worker_filter` attribute controls which workers are used for egress to a target. This is the worker
-that accesses the target.
-
-The `ingress_worker_filter`<sup>HCP Only</sup> attribute controls which workers are used for ingress to a target.
-This is the worker a client connects to when initiating a connection to a target.
```

Comment:
I believe this should also keep the `egress_worker_filter` bits for the OSS release

```
The `egress_worker_filter` attribute controls which workers are used for egress to a target. This is the worker
that accesses the target.
```